### PR TITLE
chore: add manual trigger for Polkadot releases dependencies

### PR DESCRIPTION
**Pull request by ntn-x2**
Fixes https://github.com/KILTprotocol/ticket/issues/2403.

I was playing around with `subalfred` and found out this nice feature, which we might want to integrate into our update flow. The action is configured to be run manually by providing the versions we are updating from and to.

~There is one thing that would break if we keep changing the weekly board project name to include the sprint goal, I think we could start tracking that separately.~ -> replaced `project-name` with `project-number`, thanks @ggera 

What do you guys think? Would it make sense?

### CODE REVIEW

File: .github/workflows/polkadot-release-list.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        with:
+          title: "chore: update Polkadot dependencies from ${{ inputs.current-version }} to ${{ inputs.target-version }}"
+          repository: KILTProtocol/ticket
+          token: ${{ secrets.REPO_ACCESS_TOKEN }}
+          content-filepath: out.md
+      - name: Put issue into weekly board
+        uses: peter-evans/create-or-update-project-card@v2
+        with:
+          # TODO: This will break if we change the name of the project
+          project-name: "Weekly Board 📅 Test & Release public credentials Chain 1.9.0 SDK 0.31 / CType Draft V2 / Documentation Engagement"
```
Comment (ggera) [inline .github/workflows/polkadot-release-list.yml:46]:
```suggestion
          project-number: 24
```

**Comment by ggera** [comment]
> Fixes [KILTprotocol/ticket#2403](https://github.com/KILTprotocol/ticket/issues/2403).
> 
> I was playing around with `subalfred` and found out this nice feature, which we might want to integrate into our update flow. The action is configured to be run manually by providing the versions we are updating from and to.
> 
> There is one thing that would break if we keep changing the weekly board project name to include the sprint goal, I think we could start tracking that separately.
> 
> What do you guys think? Would it make sense?

Looks awesome,, can we take a learning from [ticket-version-bump](https://github.com/KILTprotocol/ticket/blob/master/.github/workflows/version-bump-ticket.yml) and if possible put both in a single repo?

**Comment by ggera** [comment]
> Fixes [KILTprotocol/ticket#2403](https://github.com/KILTprotocol/ticket/issues/2403).
> 
> I was playing around with `subalfred` and found out this nice feature, which we might want to integrate into our update flow. The action is configured to be run manually by providing the versions we are updating from and to.
> 
> There is one thing that would break if we keep changing the weekly board project name to include the sprint goal, I think we could start tracking that separately.
> 
> What do you guys think? Would it make sense?

Looks awesome,, can we take a learning from [ticket-version-bump](https://github.com/KILTprotocol/ticket/blob/master/.github/workflows/version-bump-ticket.yml) and if possible put both in a single repo?

**Review by ggera — APPROVED** [review]

**Comment by ntn-x2** [comment]
There you go! I remember we had an action somewhere for this sort of ticket, but I could not find it!

Do you think it should live in the ticket repo or in the node repo? I personally think it should live in the node repo, which is also easier to access for most people.
Thanks for the suggestion, that's exactly what I was looking for!

**Comment by ntn-x2** [comment]
Since there's a new Polkadot 0.9.38 release, what do you think about reviewing this PR and test it with the new release? @weichweich @trusch

**Review by weichweich — APPROVED** [review]

**Pull request merged**